src/lib/controllers/aligned-controller.js

```diff
@@ -0,0 +1,57 @@
+export default class AlignedController {
+  constructor (l10n) {
+    this.l10n = l10n
+  }
+
+  createAlignedTexts (alignment) {

```

Comment:
Just a note - this is fine for now, but will need to be moved to support the use case of creating an alignment from an existing alignment  (per the sequence diagrams and the alignedtext part of the data model)

Reply:
yes, it would be in next steps